### ISSUE TITLE
[Feature Request] Find/Replace dialog: The reduce/normal mode button should keep its position

### ISSUE THREAD

**Issue by Krazal** [notepad-plus-plus/notepad-plus-plus#12615]
### Description of the Issue
It would be cool, if the reduce/normal mode button could keep its position in Find/Replace dialog, instead of sticking to the bottom right corner of the dialog.

### Expected Behavior
When I expand the Find/Replace dialog e.g. to use the Replace all button, the reduce button may disappear when the dialog is in the bottom right corner of the display. I think, it would be easier to toggle from normal to reduced mode, if the reduce button would keep its position, like the checkbox of the 2-buttons-search.

Reduced mode:
![find_replace_reduced](https://user-images.githubusercontent.com/8446933/206425808-7cbaa717-4bb5-4b78-a8b0-eb989ffce6d4.png)

Opened mode; the reduce button was disappeared:
![find_replace_opened](https://user-images.githubusercontent.com/8446933/206425853-fa857e46-9d07-4dae-89ef-e6e4384438f5.png)

Thank you for considering my suggestion. Notepad++ is the best code editor, and it's awesome, that it gets still mind-blowing new features version by version!

**Comment by Krazal** [comment]
**Workaround:** the hidden buttons are still available with accelerators (Alt + letter), so we don't need to display them to use them.

**Comment by alankilborn** [comment]
@krazal IMO, it's actually a _good_ suggestion that this button not move.  I'd like to see this issue reopened, for that reason alone (not necessarily because of any of the other stuff you said).

**Comment by Krazal** [comment]
@alankilborn True, as accelerators are not available in all languages ​​(I made the Hungarian translation, lol).
